CHANGES.rst

```diff
@@ -24,6 +24,9 @@ Specviz
 API Changes
 -----------
 
+- CLI now takes the layout as a required second positional argument
```

Comment:
Isn't it the first argument to the executable?
```suggestion
- CLI now takes the layout as a required first positional argument
```

Reply:
Technically, `jdaviz` is the command, so `layout` would be the first positional argument to the command, so your suggestion works for me. I will approve after change log is clarified. Thanks!